crates/gpui/src/window.rs

```diff
@@ -2601,7 +2601,9 @@ impl<'a> WindowContext<'a> {
             .insert_primitive(MonochromeSprite {
                 order: 0,
                 pad: 0,
-                bounds,
+                bounds: bounds
+                    .map_origin(|origin| origin.floor())
```

Comment:
We can't do it simply like this. Any change to the vertex positions must be accompanied by the change to the attributes (namely, the texture coordinates). If we are changing the positions without adjusting the texture coordinates, we are stretching the image.